CMakeLists.txt

```diff
@@ -694,7 +694,7 @@ endif()
 # Scheduler configuration
 ################################################################################
 hpx_option(HPX_WITH_THREAD_SCHEDULERS STRING
-  "Which thread schedulers are built. Options are: all, abp-priority, local, static-priority, static, shared-priority. For multiple enabled schedulers, separate with a semicolon (default: all)"
+  "Which thread schedulers are built. Options are: all, abp-priority, local, workstealing, static-priority, static, shared-priority. For multiple enabled schedulers, separate with a semicolon (default: all)"
```

Comment:
Bikeshedding that should be done before we merge this: `workstealing` isn't a very descriptive name, considering most of the other schedulers also steal work. `workrequesting` could be more appropriate.

Reply:
I like `workrequesting`!